README.md

```diff
@@ -1,2 +1,68 @@
+
+This repository contains the functionality to standardize the data of <!-- Title of checklist data or reference to source, e.g. "[Zieritz et al. (2014)](https://doi.org/10.3897/neobiota.23.5665)" --> to a [Darwin Core checklist](https://www.gbif.org/dataset-classes) that can be harvested by [GBIF](http://www.gbif.org).
+
+## Workflow
+
+<!-- This section describes how we go from raw data to standardized Darwin Core data -->
+
+[source data](data/raw) <!-- Additionally, you can write here where that raw data came from, e.g. "(downloaded as [Supplementary Material 1](http://neobiota.pensoft.net//lib/ajax_srv/article_elements_srv.php?action=download_suppl_file&instance_id=31&article_id=4007))" --> → Darwin Core [mapping script](src/dwc_mapping.Rmd) → generated [Darwin Core files](data/processed)
+
+## Published dataset
+
+<!-- This section provides links to the published dataset. Obviously, you'll only be able to add those links once you have published your dataset. 😋 -->
+
+* [Dataset on the IPT](<!-- Add the URL of the dataset on the IPT here -->)
+* [Dataset on GBIF](<!-- Add the DOI of the dataset on GBIF here -->)
+
+## Repo structure
+
+<!-- This section helps users (and probably you!) to find their way around this repository. You can leave it as is, unless you're starting to adapt the structure a lot. -->
+
+The repository structure is based on [Cookiecutter Data Science](http://drivendata.github.io/cookiecutter-data-science/) and the [Checklist recipe](https://github.com/trias-project/checklist-recipe). Files and directories indicated with `GENERATED` should not be edited manually.
+
+```
+├── README.md           : Description of this repository
+├── LICENSE             : Repository license
+├── .gitignore          : Files and directories to be ignored by git
+│
+├── data
+│   ├── raw             : Source data, input for mapping script
+│   └── processed       : Darwin Core output of mapping script GENERATED
+│
+├── docs                : Repository website GENERATED
+│
+└── src
+    ├── dwc_mapping.Rmd : Darwin Core mapping script, core functionality of this repository
+    └── src.Rproj       : RStudio project file
+```
+
+## Installation
+
+<!-- This section is for users who want to download/adapt your checklist repository. You can leave it as is. -->
+
+1. Clone this repository to your computer
+2. Open the RStudio project file
+3. Open the `dwc_mapping.Rmd` [R Markdown file](https://rmarkdown.rstudio.com/) in RStudio
+4. Install any required packages
+5. Click `Run > Run All` to generate the processed data
+6. Alternatively, click `Build > Build website` to generate the processed data and build the website in `/docs`
+
+## Contributors
+
+<!-- This section lists everyone who contributed to this repository. You can maintain a manual list here or reference the contributors on GitHub. -->
```

Comment:
I see two options:
1. we add "in case you opt for a manual list, remove the link below" or similar
2. we remove the option of a manual list and we add already a template contributor.Md with commented hidden text as here.

Reply:
I would opt for second option. Not difficult to understand and nice result at the end. People will be happy with that too.

Reply:
Is a `contributors.md` file standard GitHub practice?

Reply:
Actually no. You are right!